tests/test-team/template-mgmt-component-tests/nhs-app/template-mgmt-create-nhs-app-page.component.ts

```diff
@@ -0,0 +1,342 @@
+import { test, expect } from '@playwright/test';
+import SessionStorageHelper from '../../helpers/session-storage-helper';
+import { TemplateMgmtCreateNhsAppPage } from '../../pages/nhs-app/template-mgmt-create-nhs-app-page';
+import { SessionFactory } from '../../helpers/session-factory';
+import {
+  assertFooterLinks,
+  assertGoBackLink,
+  assertLoginLink,
+  assertNotifyBannerLink,
+  assertSkipToMainContent,
+} from '../template-mgmt-common.steps';
+
+const sessions = {
+  empty: SessionFactory.createNhsAppSession('empty-nhs-app-session'),
+  submit: SessionFactory.createNhsAppSession('submit-nhs-app-session'),
+  submitAndReturn: SessionFactory.createNhsAppSession(
+    'submit-and-return-create-nhs-app-session'
+  ),
+  goBackAndReturn: SessionFactory.createNhsAppSession(
+    'go-back-nhs-app-session'
+  ),
+  noNhsAppTemplateType: SessionFactory.create({
+    id: 'no-nhs-app-template-type-session',
+    templateType: 'UNKNOWN',
+  }),
+  previousData: {
+    ...SessionFactory.createNhsAppSession('previous-data-nhs-app-session'),
+    nhsAppTemplateName: 'previous-data-nhs-app-template',
+    nhsAppTemplateMessage: 'previous-data-nhs-app-template-message',
+  },
+};
+
+test.describe('Create NHS App template Page', () => {
```

Comment:
There is already a create template component test suite on main (here: tests/test-team/template-mgmt-component-tests/template-mgmt-create-nhs-app-template-page.component.ts), could merge the two together and remove any duplicated testing?